rdl/rdl-gen-athenz-server/main.go

```diff
@@ -480,6 +481,9 @@ func (gen *javaServerGenerator) handlerBody(r *rdl.Resource) string {
 	s += "            }\n"
 	s += "        } finally {\n"
 	s += "            this.delegate.recordMetrics(context, code);\n"
+	if r.Method == "POST" || r.Method == "PUT" || r.Method == "DELETE" {
```

Comment:
let's change the order of calling metrics and then publish, so that when we're recording metrics we also include the time we spent on publishing events.

Reply:
sure,
I am about to push new commit soon with many changes..